### PR TITLE
Require node 16 & npm 8.5; remove changelog & component status from the cookbook

### PR DESCRIPTION
**Pull request by MadsBuchmann**
## Which issue does this PR close?

This PR closes #2319 

## What is the new behavior?

I've branched out from #2319  which updates node and npm as it needed some modifications. 
Our use of firebase was outdated, but was only used for two things: 
1. The changelog 
2. The component status page 

We've decided to tear down both pages and remove related code plus dependencies. The changelog was broken anyways with our new way of doing changelogs - instead an external link has been setup which refers to [CHANGELOG.md](CHAGENLOG.md) which will act as our changelog for the future. 

The component status page was not maintained anymore and therefore we removed it. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] ~~Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".~~
- [ ] ~~Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).~~

### Review  
- [X] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [X] Request that the changes are code-reviewed 
- [ ] ~~Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)~~

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Changelog
+
+Notable changes to the project will be documented in this file.
+
+## [6.0.0](https://github.com/kirbydesign/designsystem/pull/2337) (15-06-2022)
+
+### Bugfixes
+
+**button** - change size of icon on icon-only buttons (#2306)
+**form-field** - medium sized date type inputfield layout error (#2314)
```
Comment (RasmusKjeldgaard) [inline CHANGELOG.md:10]: All the lists of features and fixes in this file lack a newline at the end:  https://github.com/kirbydesign/designsystem/blob/documentation/remove-component-status/CHANGELOG.md

Could we put them in an unordered list?
Perhaps also convert each PR reference to a link? (Quite a task - sorry if you end up doing it! 😬 )

File: libs/designsystem/src/lib/components/chart/configs/annotations.config.ts [hunk 2]
```diff
@@ -33,4 +33,6 @@ export const CHART_ANNOTATIONS_CONFIG: AnnotationTypeRegistry = {
   point: {
     backgroundColor: 'initial',
   },
+  label: { content: () => null },
+  polygon: { backgroundColor: 'initial' },
```
Comment (RasmusKjeldgaard) [inline libs/designsystem/src/lib/components/chart/configs/annotations.config.ts:37]: Are these changes safe? They stick out compared to the other objects.  

You are the chart champ after all 🙇, but I noticed these changes are from the commits before you took over this branch, so just wanted to make sure.

File: CHANGELOG.md (hunk 1 above)
Comment (MadsBuchmann) [reply CHANGELOG.md]: Fixed it!

**Review by RasmusKjeldgaard — APPROVED** [review]
Very very nice!!

**Pull request merged**